modules/tinymce/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### 5.8.1 - TBD
 
+### Added
```

Comment:
This should be in the `Unreleased` section which is what will be used for 5.9.

- [ ] Move to correct section